roles/monitoring/defaults/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+# defaults file for monitoring
+Email: xeniaspan@gmail.com
```

Comment:
edw des an mporeis na to kaneis lookup apo to env...na exeis dld mia env variable setup sto pc sou...gia na min vazeis to email sou edw sto github publicly